tests/spec/core/ui-spec.js

```diff
@@ -24,21 +24,12 @@ describe("Core - UI", () => {
     expect(window.getComputedStyle(menu).display).toBe("none");
     doc.getElementById("respec-pill").click();
     // spin the event loop
-    await new Promise(resolve => {
-      setTimeout(() => {
-        expect(window.getComputedStyle(menu).display).toBe("block");
-        doc.body.click();
-        resolve();
-      }, 500);
-    });
-    // Allow time to fade in
-    await new Promise(resolve => {
-      setTimeout(() => {
-        expect(window.getComputedStyle(menu).display).toBe("none");
-        resolve();
-      }, 500);
-    });
-    // give it time to fade out
+    await new Promise(resolve => setTimeout(resolve));
```

Comment:
should we move this as a util in SpecHelper.js?

Reply:
If we get some more use cases, sure.